
From: Alex Zarochentsev <zam@namesys.com>

no, it is not needed now.  the functions which use get_full_page_state()
are not used anywhere else.

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/fs/reiser4/page_cache.c |   66 ----------------------------------------
 25-akpm/fs/reiser4/page_cache.h |    4 --
 2 files changed, 70 deletions(-)

diff -puN fs/reiser4/page_cache.c~reiser4-remove-debug fs/reiser4/page_cache.c
--- 25/fs/reiser4/page_cache.c~reiser4-remove-debug	Thu Aug 19 15:22:14 2004
+++ 25-akpm/fs/reiser4/page_cache.c	Thu Aug 19 15:22:14 2004
@@ -903,72 +903,6 @@ print_page(const char *prefix, struct pa
 	}
 }
 
-reiser4_internal void
-print_page_state(const char *prefix, struct page_state *ps)
-{
-	printk("%i: %s: "
-	       "free: %u, "
-	       "dirty: %lu, "
-	       "writeback: %lu, "
-//	       "pagecache: %lu, "
-//	     "page_table_pages: %lu, "
-//	     "reverse_maps: %lu, "
-	       "mapped: %lu, "
-	       "slab: %lu, "
-//	     "pgpgin: %lu, "
-//	     "pgpgout: %lu, "
-//	     "pswpin: %lu, "
-//	     "pswpout: %lu, "
-//	     "pgalloc: %lu, "
-//	     "pgfree: %lu, "
-//	     "pgactivate: %lu, "
-//	     "pgdeactivate: %lu, "
-//	     "pgfault: %lu, "
-//	     "pgmajfault: %lu, "
-//	     "pgscan: %lu, "
-//	     "pgrefill: %lu, "
-//	     "pgsteal: %lu, "
-	       "kswapd_steal: %lu, "
-//	     "pageoutrun: %lu, "
-//	     "allocstall: %lu
-	       "\n", current->pid, prefix,
-
-	       nr_free_pages(),
-	       ps->nr_dirty,
-	       ps->nr_writeback,
-//	       ps->nr_pagecache,
-//	     ps->nr_page_table_pages,
-//	     ps->nr_reverse_maps,
-	       ps->nr_mapped,
-	       ps->nr_slab,
-//	     ps->pgpgin,
-//	     ps->pgpgout,
-//	     ps->pswpin,
-//	     ps->pswpout,
-//	     ps->pgalloc,
-//	     ps->pgfree,
-//	     ps->pgactivate,
-//	     ps->pgdeactivate,
-//	     ps->pgfault,
-//	     ps->pgmajfault,
-//	     ps->pgscan,
-//	     ps->pgrefill,
-//	     ps->pgsteal,
-	       ps->kswapd_steal //,
-//	     ps->pageoutrun,
-//	     ps->allocstall
-		);
-}
-
-reiser4_internal void
-print_page_stats(const char *prefix)
-{
-	struct page_state ps;
-	get_full_page_state(&ps);
-	print_page_state(prefix, &ps);
-}
-
-
 #endif
 
 /* Make Linus happy.
diff -puN fs/reiser4/page_cache.h~reiser4-remove-debug fs/reiser4/page_cache.h
--- 25/fs/reiser4/page_cache.h~reiser4-remove-debug	Thu Aug 19 15:22:14 2004
+++ 25-akpm/fs/reiser4/page_cache.h	Thu Aug 19 15:22:14 2004
@@ -50,12 +50,8 @@ extern void capture_reiser4_inodes (stru
 
 #if REISER4_DEBUG_OUTPUT
 extern void print_page(const char *prefix, struct page *page);
-extern void print_page_state(const char *prefix, struct page_state *ps);
-extern void print_page_stats(const char *prefix);
 #else
 #define print_page(prf, p) noop
-#define print_page_state(prefix, ps) noop
-#define print_page_stats(prefix) noop
 #endif
 
 /* __REISER4_PAGE_CACHE_H__ */
_
